src/runway/config.py

```diff
@@ -281,19 +280,6 @@ class DeploymentDefinition(ConfigComponent):  # pylint: disable=too-many-instanc
               "*":  # applied to all environments
                 ANOTHER_VAR: foo
 
-    A deployment can be defined without modules if the directory
-    containing the runway config file is a module directory.
-
-    Example:
-      .. code-block:: yaml
-
-        deployments:
-          - current_dir: true
-            regions:
-              - us-west-2
-            assume-role:
-              arn: arn:aws:iam::0000:role/role-name
-
```

Comment:
We should describe the new `modules: ['./']` design pattern in place of this section. It may not be as intuitive to some people as `current_dir` was.